source/reference/command/find.txt

```diff
@@ -74,6 +74,11 @@ The :dbcommand:`find` command has the following syntax:
       }
    )
 
+.. note::
+  Since MongoDB 4.4, the :dbcommand:`find` command ignores the
```

Comment:
[Minor]

Generally we don't need to link to the page that the reader is currently on. We can just monospace `find` here.

```suggestion
  Since MongoDB 4.4, the ``find`` command ignores the
```